### DISCUSSION TITLE
Does sesh support persistence across reboots or integrate with tmux-resurrect?

### DISCUSSION THREAD

**Discussion by mloskot** [joshmedeski/sesh#82] [General]
I haven't tried the `sesh` yet, but from its docs it is not entirely clear to me if it does support any form of saving/restoring sessions including running programs, and does it across reboots. Does `sesh` integrate with https://github.com/tmux-plugins/tmux-resurrect or https://github.com/jimeh/tmuxifier, or offer an alternative/built-in functionality that does the same?

**Comment by joshmedeski** [comment]
Sesh does not currently support restoring sessions. There are scripts that can scaffold projects if you have certain panes or windows you want to open every time you start a project, so look into that feature on the README.

**Reply by mloskot** [reply]
Thank you!

**Comment by laur89** [comment]
@mloskot curious whether you ended up using `sesh`, and which way you took to solve the session persistence?

**Reply by mloskot** [reply]
@laur89 Yes, I'm still using `sesh` and I ended up relying on `startup_command` and scripts, see my [sesh.toml](https://github.com/mloskot/dotfiles/blob/ba82fa245579c815e0c9df890978a6153bba6e16/config/sesh/sesh.toml).

I'm also exploring Zellij as alternative to tmux-based sessions, especially after Zellij sessions persistence received the [action save-session](https://github.com/zellij-org/zellij/pull/4654) completing the [session management features](https://zellij.dev/documentation/cli-recipes.html#session-management).

**Reply by laur89** [reply]
Cheers. If I'm reading it right, it's still static scripted config as opposed to anything like resurrect/continuum?

**Reply by mloskot** [reply]
@laur89 Yes, correct, with `sesh` I'm using scripts with static pre-defined sessions. I only use the real resurrection/continuum mode with the Zellij, but I haven't learned if it's feasible/sensible/how to integrate it with `sesh`.

**Reply by laur89** [reply]
I suppose using sesh _thogether with_ resurrect/continuum shouldn't be mutually exclusive either. Use sesh to actively open/close/manage sessions, whereas letting resurrect+continuum to back up currently opened sessions & restoring them on tmux server start.
